services/imgur.yml

```diff
@@ -0,0 +1,5 @@
 10.840953 35.316063 11.313063 34.837094 11.914062 34.621094 C 12.028062 34.580094 12.027109 34.415047 11.912109 34.373047 C 11.312109 34.160047 10.839 33.685938 10.625 33.085938 C 10.605 33.028937 10.553578 33 10.501953 33 z M 29.001953 33 C 28.967453 33 28.933422 33.018641 28.919922 33.056641 C 28.775922 33.456641 28.455641 33.774016 28.056641 33.916016 C 27.980641 33.943016 27.980641 34.053078 28.056641 34.080078 C 28.455641 34.224078 28.775922 34.544359 28.919922 34.943359 C 28.947922 35.019359 29.056984 35.019359 29.083984 34.943359 C 29.225984 34.544359 29.543359 34.224078 29.943359 34.080078 C 30.019359 34.053078 30.019359 33.943016 29.943359 33.916016 C 29.543359 33.774016 29.225984 33.456641 29.083984 33.056641 C 29.070484 33.018641 29.036453 33 29.001953 33 z M 26.001953 36 C 25.933078 35.999875 25.865391 36.037281 25.837891 36.113281 C 25.550891 36.914281 24.911281 37.547031 24.113281 37.832031 C 23.960281 37.887031 23.961281 38.107109 24.113281 38.162109 C 24.912281 38.450109 25.549891 39.088719 25.837891 39.886719 C 25.892891 40.038719 26.113969 40.036766 26.167969 39.884766 C 26.452969 39.086766 27.085719 38.450109 27.886719 38.162109 C 28.038719 38.108109 28.037766 37.887031 27.884766 37.832031 C 27.083766 37.547031 26.452969 36.914281 26.167969 36.113281 C 26.140969 36.036781 26.070828 36.000125 26.001953 36 z M 9.0019531 38 C 8.9674531 38 8.9334219 38.018641 8.9199219 38.056641 C 8.7759219 38.456641 8.4556406 38.774016 8.0566406 38.916016 C 7.9806406 38.943016 7.9806406 39.053078 8.0566406 39.080078 C 8.4556406 39.224078 8.7759219 39.544359 8.9199219 39.943359 C 8.9479219 40.019359 9.0569844 40.019359 9.0839844 39.943359 C 9.2259844 39.544359 9.5433594 39.224078 9.9433594 39.080078 C 10.019359 39.053078 10.019359 38.943016 9.9433594 38.916016 C 9.5433594 38.774016 9.2259844 38.456641 9.0839844 38.056641 C 9.0704844 38.018641 9.0364531 38 9.0019531 38 z M 13.501953 39 C 13.450328 39 13.399406 39.028937 13.378906 39.085938 C 13.163906 39.686938 12.684938 40.159047 12.085938 40.373047 C 11.971938 40.413047 11.971937 40.580094 12.085938 40.621094 C 12.684937 40.837094 13.162906 41.315063 13.378906 41.914062 C 13.419906 42.028062 13.585953 42.028062 13.626953 41.914062 C 13.840953 41.316063 14.313063 40.837094 14.914062 40.621094 C 15.028062 40.580094 15.027109 40.415047 14.912109 40.373047 C 14.312109 40.160047 13.839 39.685938 13.625 39.085938 C 13.605 39.028937 13.553578 39 13.501953 39 z M 20.501953 40 C 20.450328 40 20.399406 40.028938 20.378906 40.085938 C 20.163906 40.686938 19.684938 41.159047 19.085938 41.373047 C 18.971937 41.413047 18.971937 41.580094 19.085938 41.621094 C 19.684938 41.837094 20.162906 42.315063 20.378906 42.914062 C 20.419906 43.028062 20.585953 43.028062 20.626953 42.914062 C 20.840953 42.316062 21.313063 41.837094 21.914062 41.621094 C 22.028062 41.580094 22.027109 41.415047 21.912109 41.373047 C 21.312109 41.160047 20.839 40.685938 20.625 40.085938 C 20.605 40.028937 20.553578 40 20.501953 40 z"/></svg>
```

Comment:
for example, `www.w3.org/2000/svg"  viewBox="0 0 50 50">` has redundant spaces, maybe we should format this property later

Reply:
@slavaleleka maybe we should take a look at https://github.com/svg/svgo

Reply:
maybe for another time. let's add todo for now